### PR TITLE
Provide support script for adding new test subject

### LINKED ISSUE #816: [Feature Request] Provide support scripts for parties interested in submitting test reports
While [writing documentation](https://github.com/SovereignCloudStack/docs/issues/271), I'm realizing that some of the steps NEED to be automated with simple support scripts, for instance:

- create keypair for signing the reports
- create token for accessing the compliance-monitor API
- move secrets into proper places
- etc. pp.

### PR DESCRIPTION
**Pull request by mbuechse**
Resolves #816

### CODE REVIEW

File: Tests/add_subject.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    print("Creating API key...")
+    while True:
+        password = getpass.getpass("Enter passphrase: ")
+        if password == getpass.getpass("Repeat passphrase: "):
+            break
+        print("No match. Try again...")
+    token = base64.b64encode(f"{subject}:{password}".encode('utf-8'))
+    hash_ = CRYPTCTX.hash(password)
+    with open(tokenfile_path, "wb") as fileobj:
+        fileobj.write(token)
```
[the remark this answers is not in the document]
Comment (mbuechse) [reply Tests/add_subject.py:53]: Erm what? And the `key`? Where am I storing the key???

[the remark this answers is not in the document]
Comment (garloff) [reply Tests/add_subject.py:53]: Without spending an hour to understand what's required here, I can not comment on this, sorry.
If we can avoid storing sensitive data in plaintext, we should of course. I have no idea whether or not that is possible without major effort. If we need to store it, we should adjust the permissions / umask to 0600 (rw-------).

**Comment by mbuechse** [comment]
@garloff Your suggestion with the access rights is fine from my POV. The secrets need to be accessed without user intervention, but they are not terribly critical -- yes, an attacker who snitches them could upload bad (forged) test results for the CSP, but we always check failed tests, and we can easily disable/rotate the keys.

**Review by garloff — APPROVED** [review]
This works for me -- we can always improve later, but this is good enough for now.

**Pull request merged**